app/frontend/src/search/ui/sort.js

```diff
@@ -1,38 +1,25 @@
 export const renderSortSelect = (renderOptions, isFirstRender) => {
-  const {
-    options,
-    currentRefinement,
-    hasNoResults,
-    refine,
-    widgetParams,
-  } = renderOptions;
+  const { hasNoResults, refine, widgetParams } = renderOptions;
 
   if (isFirstRender) {
-    const select = document.createElement('select');
-    select.classList.add('govuk-select');
-    select.classList.add('govuk-input--width-10');
-    select.id = 'jobs_sort_select';
+    if (document.querySelector('#submit_job_sort')) {
+      document.querySelector('#submit_job_sort').style.display = 'none';
+    }
 
-    select.addEventListener('change', (event) => {
-      if (document.querySelector('ul.vacancies')) {
-        document.querySelector('ul.vacancies').style.display = 'none';
-      }
-      refine(event.target.value);
-    });
-
-    widgetParams.container.appendChild(select);
-  }
-
-  if (document.getElementById('jobs_sort_label')) {
-    document.getElementById('jobs_sort_label').style.display = hasNoResults ? 'none' : 'inline-block';
+    if (document.querySelector('#jobs_sort_select')) {
+      document.querySelector('#jobs_sort_select').addEventListener('change', (event) => {
+        refine(constructSearchReplica(event.target.value));
+      });
+    }
   }
 
-  const select = document.getElementById('jobs_sort_select');
-  if (select) {
-    select.style.display = hasNoResults ? 'none' : 'inline-block';
+  widgetParams.container.style.display = hasNoResults ? 'none' : 'inline-block';
 
-    select.innerHTML = constructOptions(options, currentRefinement);
+  if (hasNoResults) {
+    widgetParams.element.disabled = true;
+  } else {
+    widgetParams.element.removeAttribute('disabled');
   }
 };
 
-export const constructOptions = (options, selected) => `${options.map((option) => `<option value="${option.value}"${option.value === selected ? ' selected' : ''}>${option.label}</option>`).join('')}`;
+export const constructSearchReplica = (selected) => (selected ? `Vacancy_${selected}` : 'Vacancy');
```

Comment:
Could we rename `constructSearchReplica` to something like `getSearchReplicaName` or `makeSearchReplicaName`, for extra clarity?

Reply:
Hell, even `getSearchReplicaIndexName`